src/globals/internal/FloatingMenu.js

```diff
@@ -308,7 +316,8 @@ class FloatingMenu extends React.Component {
             {
               left: `${pos.left}px`,
               top: `${pos.top}px`,
-              position: 'relative',
+              bottom: 'unset', // dont know if we can/should remove this
```

Comment:
this part is a workaround for the stylesheet specifying `bottom: 1rem`, which doesn't play nicely with the absolute positioning (the tooltip stretches the whole page if this line is omitted)

Reply:
Would also make sense to remove the bottom value from SASS files like fileUploader's tooltip